client/utils/trpc/trpc-client.utils.ts

```diff
@@ -1,6 +1,7 @@
-import { createTRPCProxyClient, httpBatchLink } from "@trpc/client";
+import { createTRPCProxyClient, httpBatchLink, TRPCLink } from "@trpc/client";
 import type { AppRouter } from "@/server/routers/_app";
 import superjson from "superjson";
+import { observable } from "@trpc/server/observable";
```

Comment:
Is this import right? Just wondering, because it says "server" and this file is for the client.

Reply:
Yes, its says server but works fine. Look at this `httpBatchLink` also has imported from server: https://github.com/trpc/trpc/blob/next/packages/client/src/links/httpBatchLink.ts